app.js

```diff
@@ -101,7 +98,7 @@ app.post('/api/request', function (req, res) {
       'version-specberus': SpecberusWrapper.version,
       decision: decision,
       state: new RequestState(
-        STATUS_STARTED,
+        '',
```

Comment:
I know, this is not pretty, but it will evolve: things will move from `app.js` to `lib` components...